package.json

```diff
@@ -16,7 +16,7 @@
     "classnames": "^2.2.6",
     "graphql": "^14.5.8",
     "i18next": "^17.0.18",
-    "node-sass": "^4.12.0",
+    "node-sass": "^4.13.0",
```

Comment:
Was this needed to create the Dockerfile?

If not, then this change should be in another PR.

Reply:
This was actually needed to create the Dockerfile. Something related to node12/node-gyp/node-sass dependency alignment... I'll double check to be sure.

Reply:
This actually isn't needed anymore since the container has build-essentials.